application/internal/test/test.go

```diff
@@ -175,7 +177,7 @@ func CreateFileFixture() models.File {
 		Name:    strconv.Itoa(rand.Int()) + ".gif",
 		Content: []byte("GIF89a"),
 	}
-	if err := f.Store(); err != nil {
+	if err := f.Store(models.DB); err != nil {
```

Comment:
is this right? It's a test so maybe using global connection is okay?

Reply:
Probably, but I'll pass in the connection anyway.